toolbox/include/DiscreteFilter.h

```diff
@@ -0,0 +1,47 @@
+#include "Block.h"
+#include <string>
+#include <vector>
+
+#ifndef WBT_FILTER_H
+#define WBT_FILTER_H
+
+namespace wbt {
+    class DiscreteFilter;
+} // namespace wbt
+
+namespace iCub {
+    namespace ctrl {
+        class IFilter;
+    }
+} // namespace iCub
+
+namespace yarp {
+    namespace sig {
+        class Vector;
+    }
+} // namespace yarp
+
+class wbt::DiscreteFilter : public wbt::Block {
+private:
+    bool firstRun;
+    iCub::ctrl::IFilter* filter;
+    yarp::sig::Vector* num;
```

Comment:
Why putting `num` and `den` in the `.h` file? They are only needed in configuration and not in real time.